packages/connect-explorer/src/pages/methods/other/init.mdx

```diff
@@ -33,14 +42,27 @@ export const paramDescriptions = {
     popup: 'default `true`. Projects running on trusted domains (trezor.io) are not required to use popup. For other domains this option is ignored.',
     transports:
         'Array of transports that should be use. If not provided, TrezorConnect will chose a reasonable default based on your environment.',
+    coreMode:
+        'default `iframe`. If set to `popup`, TrezorConnect will use popup for interaction with user. The advantage is that popup enables WebUSB to work cross-origin on the web. The disadvantage is that popup does not retain a stateful connection to the device, which can be a problem for some use-cases. If set to `auto` TrezorConnect will use iframe only if bridge is available.',
```

Comment:
maybe @Hannsek check on this. "will use popup for interaction with user" feels somewhat misleading to me, it will use popup regardless of the mode. The rests sounds good to me.

Reply:
I just updated the text slightly

Reply:
I think that the naming "popup" itself is misleading a bit. If we won't change the naming I'd do something like:

If set to `popup`, webUSB transport would work cross-origin on the web. The disadvantage of this mode is that Connect popup does not retain a stateful connection to the device. 

If set to `auto`, TrezorConnect will use iframe with persistent connection for the communication between popup and 3rd party wallet only if nodeBridge is available.